specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -5230,20 +5230,23 @@
         "keyVaultKeyVersion",
         "keyVaultUri"
       ],
-      "description": "The configuration to use encryption with customer-managed keys. This information will be used to encrypt or decrypt the data you have protected using your own keys."
+      "description": "A fully-specified, user-defined encryption key in Azure Key Vault. Keys that you create and manage can be used to encrypt or decrypt data-at-rest in Azure Search, such as indexes and synonym maps."
     },
     "AzureActiveDirectoryApplicationCredentials": {
       "properties": {
         "applicationId": {
           "type": "string",
-          "description": "The is the ID of the Azure Active Directory Application that will be used to authenticate to your KeyVault. The Application ID is not to be confused with the Object ID of your Azure Active Directory Application."
+          "description": "The Application ID of an Azure Search service, when registered as an Azure Active Directory application, for the purpose of serving as a security principle for authentication to Azure Key Vault. The Application ID should not be confused with the Object ID for your application. Both IDs are GUIDs but have different purposes. The Application ID uniquely identifies the app on protocol transactions."
         },
         "applicationSecret": {
           "type": "string",
-          "description": "The application secret, also sometime refered to as the authentication key value. This will be used to authenticate to your KeyVault."
+          "description": "The application secret, also referred to as the authentication key value. This is the actual key used to authenticate to Azure Key Vault. When creating a key for a registered application, this is the string that is given when you save the key. This value is not visible in the portal after you leave the blade."
```

Comment:
It seems weird to be talking about the Portal experience for managing AAD keys from API documentation.

@HeidiSteen What's the goal of including it here? Can we find a way to achieve the same thing while keeping things in the realm of the API and doing things programmatically?

Reply:
Ok, disregard the proposed text.  I added it because there are multiple IDs/values in the UI and in the docs, and the a-ha mometn for me was seeing it in the portal, on that blade that says "copy now because you will never see it again"